homeassistant/components/emoncms/__init__.py

```diff
@@ -1 +1,32 @@
 """The emoncms component."""
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_URL
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN, LOGGER
+
+PLATFORMS: list[str] = ["sensor"]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
```

Comment:
We should use `__init__.py` to check if we can connect to the service and if we can continue setting up the integration